imgresize.cpp

```diff
@@ -0,0 +1,95 @@
+// cpp program to read,display and resize the image to our desired dimensions using opencv. 
+#include <opencv2/opencv.hpp>
+#include <opencv2/highgui.hpp>
+#include <iostream>
+
+using namespace std;
+using namespace cv;
+// declaraing global variables(so that they can further be used in any of the function).
+// Mat-The Basic Image Container.
+Mat img;
+// the header name which you want to display on the top of image widow.
+char windowname[50] = "Image";
+// function to resize the image.It is of void type,as it returns nothing.
+void imgresize()
+{
+    // these variables aur used to store the dimensions of new image
+    double w,h;
+    // to store the dimensions of image
+    Size s = img.size();
+    cout<< "Resize Image." <<endl;
+    cout<< "Width: ";
+    cin>> w;
+    cout<< "Height: ";
+    cin>> h;
```

Comment:
Let's use `cout <<` and `cin >>` for consistency instead of `cout<<` and `cin>>`.